### ISSUE TITLE
(1.12.1) Conflict with combination of mods 0-1 fps.

### ISSUE THREAD

**Issue by BubbleVanilla** [ForestryMC/ForestryMC#1803]
Server launched just fine, but after joining my own server (with this mod in both client and the server) , I can barely move, but it does not crash in any way and server did not report anything.
 Problem found - Not enough memory, allocated more ram for MC and everything is back to normal.
Other Mods installed :

SimpleCorn
GlobalXP
Industrial Wires
ImmersiveEngineering
IC2
CodeChickenLibrary
RFTools
Redstoneflux
MCJYlib
InteractiveWheel
TheOneProbe
Baubles
JEI
Steve's Carts 2
ToolBelt
RefineStorage
RebornCore
AppleSkin
RebornStorage
Forgelin
ShadowMC
Extra Rails

Update 1: I gave it a try on single player, and the fps was 0 to 1, and I had to force close it.
Update 2: I realized this is client side and not server.
Update 3: Removed All other mods and it works just fine, now I'm assuming theres a mod conflict not mentioned anywhere. I wonder what was it?
Update 4: After testing the mods combination one by one, found out that it conflict with Immersive Engineering.
Update 5: Correction, it was a combination of some mods that are causing it, not IE specifically.
Update 6: Problem found - Not enough memory, allocated more ram for MC and everything is back to normal.

**Comment by mezz** [comment]
What did you find?

**Comment by BubbleVanilla** [comment]
Mistake on my side, it wasnt Immersive Engineering that was causing the lag. At first, I tried adding mods one by one into the folder and everything works until I add immersive engineering in it.

Later I tried with JUST Forestry, IC2 and Immersive Engineering and it works just fine, so I'm guessing it wasn't Immersive Engineering, it was something in combination of those mods listed that are causing it that I couldn't figure out.
